inference-engine/tests/functional/plugin/shared/src/base/import_export_base/import_export_base.cpp

```diff
@@ -26,6 +27,8 @@ std::string ImportNetworkTestBase::getTestCaseName(testing::TestParamInfo<export
         result << "_importConfigItem=" << configItem.first << "_" << configItem.second;
     }
     result << "_appHeader=" << appHeader;
+    result << CommonTestUtils::vec2str(inputShape);
```

Comment:
Maybe add "_" separator?

Reply:
reverted this change and used the old test.